### DISCUSSION TITLE
How to start system service without a command line interface? (Windows)

### DISCUSSION THREAD

**Discussion by bulletproof2k** [autobrr/autobrr#962] [Q&A] [answered]
I created a task to start at log on, but I don't really like to see and then hide the command line interface in the taskbar.

**Comment by Yankees4life** [comment]
Nssm?

**Reply by bulletproof2k** [reply]
I will try 🤝

**Comment by bulletproof2k** [comment, marked as answer]
Using the system service is not always convenient, I found a simple solution. https://github.com/stbrenner/SilentCMD
